server/app/com/xsn/explorer/models/persisted/Transaction.scala

```diff
@@ -16,17 +16,25 @@ object Transaction {
       fromOutputIndex: Int,
       index: Int,
       value: BigDecimal,
-      addresses: List[Address]
+      addresses: List[Address],
+      script: Option[HexString]
   ) {
 
     def address: Option[Address] = addresses.headOption
   }
 
   object Input {
 
-    def apply(fromTxid: TransactionId, fromOutputIndex: Int, index: Int, value: BigDecimal, address: Address): Input = {
+    def apply(
+        fromTxid: TransactionId,
+        fromOutputIndex: Int,
+        index: Int,
+        value: BigDecimal,
+        address: Address,
+        script: Option[HexString]
```

Comment:
This method exists for compatibility purposes, before, there was a single address stored, avoid updating this method and you won't need to update that many files.